classes/SpecUploadBase.php

```diff
@@ -984,8 +984,13 @@ protected function transferOccurrences(){
 					$insertCnt = $this->conn->affected_rows;
 					$warnCnt = $this->conn->warning_count;
 					if($warnCnt){
-						if(strpos($this->conn->get_warnings()->message,'UNIQUE_occurrenceID'))
```

Comment:
This condition needs to stay as it addresses a specific scenario when users are attempting to upload records from another collection